usr/scripts/detectionOn.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# Source your custom motion configurations
+. /usr/config/motion.conf
+. /usr/scripts/common_functions.sh
+
+# Turn on the amber led
+if [ "$motion_trigger_led" = true ] ; then
+	yellow_led on
+fi
+
+# Save a snapshot
+if [ "$save_snapshot" = true ] ; then
+	filename=$(date +%d-%m-%Y_%H.%M.%S).jpg
+	if [ ! -d "$save_dir" ]; then
+		mkdir -p "$save_dir"
+	fi
+	# Limit the number of snapshots
+	if [ "$(ls "$save_dir" | wc -l)" -ge "$max_snapshots" ]; then
+		rm -f "$save_dir/$(ls -l "$save_dir" | awk 'NR==2{print $9}')"
+	fi
+	/usr/bin/getimage > "$save_dir/$filename" &
+fi
+
+# Publish a mqtt message
+if [ "$publish_mqtt_message" = true ] ; then
+	. /usr/config/mqtt.conf
+	/usr/bin/mosquitto_pub.bin -h "$HOST" -p "$PORT" -u "$USER" -P "$PASS" -t "${TOPIC}"/motion ${MOSQUITTOOPTS} ${MOSQUITTOPUBOPTS} -m "ON"
+	if [ "$save_snapshot" = true ] ; then
+		/usr/bin/mosquitto_pub.bin -h "$HOST" -p "$PORT" -u "$USER" -P "$PASS" -t "${TOPIC}"/motion/snapshot ${MOSQUITTOOPTS} ${MOSQUITTOPUBOPTS} -f "$save_dir/$filename"
+	fi
+
+fi
+
+# Send emails ...
+if [ "$sendemail" = true ] ; then
+    /usr/scripts/sendPictureMail.sh&
+fi
+
+# Send a telegram message
+if [ "$send_telegram" = true ]; then
+	if [ "$save_snapshot" = true ] ; then
+		/usr/bin/telegram p "$save_dir/$filename"
+	else
+		/usr/bin/getimage > "telegram_image.jpg"
+ +	/usr/bin/telegram p "telegram_image.jpg"
+ +	rm "telegram_image.jpg"
+	fi
+fi
+
+# Run any user scripts.
+for i in /usr/config/userscripts/motiondetection/*; do
```

Comment:
I have userscripts in /usr/userscripts.
Not sure where user changable scripts would go on a normal linux system.
Probably /usr/local/bin ?

Reply:
Don't know either
we can choose what we want ...

Reply:
```/usr/local/bin``` seems the correct place. And many users will be already familiar with it.

👍 for ```/usr/local/bin```

Reply:
Go for:
for i in/usr/local/bin/motiondetection/*; do ?

Reply:
+1